modules/test/playwright/pages/object/modelBuilder.page.ts

```diff
@@ -53,15 +63,16 @@ export class ModelBuilderPage {
 	async clickObjectDefinitionShowAllFieldsButton(
 		objectDefinitionName: string
 	) {
+		await this.makeAllObjectDefinitionNodesVisible();
+
 		await this.objectDefinitionNodes
 			.filter({hasText: objectDefinitionName})
 			.getByRole('button', {name: 'Show All Fields'})
 			.click();
 	}
 
-	async goto() {
-		await this.objectDefinitionsPage.goto();
-		await this.objectDefinitionsPage.viewInModelBuilder();
+	async clickToggleSidebarsButton() {
+		this.toggleSidebarsButton.click();
```

Comment:
@cgmonte just create a new method only if:
1 - It's called outside the POM
2 - It's a complex call and used more than once inside the POM.